src/Compilers/CSharp/Portable/BoundTree/BoundNode.cs

```diff
@@ -22,20 +22,23 @@ private enum BoundNodeAttributes : byte
         {
             HasErrors = 1 << 0,
             CompilerGenerated = 1 << 1,
-            TopLevelNullableUnset = 0,
-            TopLevelNullable = 1 << 2,
-            TopLevelNonNullable = 1 << 3,
-            TopLevelUnknown = TopLevelNullable | TopLevelNonNullable,
-            TopLevelNullabilityMask = TopLevelUnknown,
+            IsSuppressed = 1 << 2,
+
+            // Bit 3: 1 if the node is has maybe-null state, 0 if the node is not null
+            // Bits 4 and 5: 01 if the node is not annotated, 10 if the node is annotated, 11 if the node is disabled
+            TopLevelFlowStateNullable = 1 << 3,
+            TopLevelNotAnnotated = 1 << 4,
+            TopLevelAnnotated = 1 << 5,
+            TopLevelDisabled = TopLevelAnnotated | TopLevelNotAnnotated,
```

Comment:
>Disabled [](start = 20, length = 8)

Consider using `Unknown` (or `Oblivious` if that's the current term) to match `NullableAnnotation`.

Reply:
Disabled is the current term. I'll update the public API.

---
In reply to: [265668513](https://github.com/dotnet/roslyn/pull/34017#discussion_r265668513) [](ancestors = 265668513)